tests/storages_tests/test_journal.py

```diff
@@ -23,19 +23,14 @@
 from optuna.testing.tempfile_pool import NamedTemporaryFilePool
 
 
-LOG_STORAGE = {
+LOG_STORAGE = [
     "file_with_open_lock",
     "file_with_link_lock",
     "redis_default",
     "redis_with_use_cluster",
-}
+]
```

Comment:
The order of `set` items is not deterministic. As a result, `pytest-xdist` recognizes the collected tests are different between workers.